src/utils/getCollectionMetadata.tsx

```diff
@@ -0,0 +1,49 @@
+import { EN_DASH } from "@/constants/chars";
+import { IMetadata, TCollectionPublicWithFamilies } from "@/types";
+import { buildConceptHierarchy } from "@/utils/buildConceptHierarchy";
+import { displayConceptHierarchy } from "@/utils/displayConceptHierarchy";
+
+export const getCollectionMetadata = (collection: TCollectionPublicWithFamilies): IMetadata[] => {
+  const collectionMetadata: IMetadata[] = [];
+
+  // Get all events from all families in the collection
+  const allEvents = collection.families.flatMap((family) => family.events);
+
+  const filingDates = allEvents
+    // TODO: confirm if events must be of type "Filing Year For Action" or "Filing Date"
+    // .filter((event) => event.event_type === "Filing Year For Action" || event.event_type === "Filing Date")
+    .map((event) => new Date(event.date));
+
+  if (filingDates.length > 0) {
+    const earliestFilingDate = new Date(Math.min(...filingDates.map((date) => date.getTime())));
+    collectionMetadata.push({
+      label: "Filing date",
+      value: earliestFilingDate ? earliestFilingDate.getFullYear() : EN_DASH,
+    });
+  }
+
+  collectionMetadata.push({
+    label: "Status",
+    value: collection.families[0]?.metadata?.status || EN_DASH,
+  });
+
+  collectionMetadata.push({
+    label: "At issue",
+    value: collection.description,
+  });
+
```

Comment:
question: This looks OK for litigation - assuming this will be more abstracted when doing collections outside of cases?

Might be worth us looking at the API here as the events and the like are a little unclear I think.